Resources/config/form.config.xml

```diff
@@ -10,21 +10,21 @@
             <argument type="service" id="integrated_channel.form.config.type" />
             <argument type="string">channel_config_new</argument>
 
-            <tag name="form.type" alias="channel_config_new" />
+            <tag name="form.type" />
```

Comment:
Can you see if the parameter can be removed and put just punt inside the class attribute of the service definition